src/app/services/object-loader/loaderTypes.ts

```diff
@@ -61,4 +72,33 @@ export interface CubeMap {
   nz: string;
 }
 
+export class CubeMap implements ICubeMap {
+  constructor(name: string, path: string) {
+    this.name = name;
+    this.path = path;
+    this.tex = undefined;
+    this.px = 'posx.jpg';
+    this.py = 'posy.jpg';
+    this.pz = 'posz.jpg';
+    this.nx = 'negx.jpg';
+    this.ny = 'negy.jpg';
+    this.nz = 'negz.jpg';
+  }
+
+  name: string;
+  nx: string;
+  ny: string;
+  nz: string;
+  path: string;
+  px: string;
+  py: string;
+  pz: string;
+  tex: CubeTexture;
+}
+
+export interface Prop<T> {
```

Comment:
if not used, please remove

Reply:
i did

Reply:
check again. there are no changes in loaderTypes.ts so far